.github/ISSUE_TEMPLATE/1_Bug_Report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Report a bug in ARTIQ    
+
+---
+
+<!-- Above are non-Markdown tags for Github auto-prompting issue type. Template based on pylint: https://raw.githubusercontent.com/PyCQA/pylint/master/.github/ISSUE_TEMPLATE/ -->
+
+# Bug Report
+
+<!-- Thanks for reporting a bug report to ARTIQ! You can also discuss issues and ask questions on IRC (the [#m-labs channel on freenode](https://webchat.freenode.net/?channels=m-labs) or on the [ARTIQ mailing list](https://ssl.serverraum.org/lists/listinfo/artiq). Please check Github/those forums to avoid posting a repeat issue.  
+
+Context helps us fix issues faster, so please include the following: 
+-->
+
+## One-Line Summary
+
+Short summary.
+
+## Issue Details
+
+### Steps to Reproduce
+
+1. Step 1.
+2. Step 2.
+3. Step 3.
+
+### Expected Behavior
+
+Behavior
+
+### Actual (undesired) Behavior
+
+* Text description
+* Log message, tracebacks, screen shots where relevant
+
+### Your System
+
+* Operating System:
+* Conda version:
+* ARTIQ version: (package or git commit id, versions for bitstream, BIOS, runtime and host software). `conda list`
```

Comment:
It's called "bootloader" these days.

Reply:
What is called "bootloader"? the bitstream? Or runtime software?

Reply:
Fixed in https://github.com/m-labs/artiq/pull/1163/commits/b072f7eee339bcd5687f28e3b05396e623b0d834